node/utils/templates/templateMessages.ts

```diff
@@ -0,0 +1,2 @@
nt;">\n                            Condition</h3>\n                        <table style="border-collapse:collapse;font-size:14px;width: 36rem">\n                            <tr>\n                                <td>\n                                    <div>\n                                        {{#if timeline}}\n                                        {{#each timeline}}\n\n                                        <div>\n                                            <p class="statusline">\n                                                {{#compare active \'==\' 1}}\n                                                <span class="statusIcon statusIconChecked">\n                                                        {{else}}\n                                                            <span class="statusIcon">\n                                                        {{/compare}}\n                                                            {{#compare active \'==\' 1}}\n                                                                <span style="color: #fff; display: block; width: 1.3rem; height: 1.3rem; text-align: center;"><b>✓</b></span>\n                                                            {{/compare}}\n                                                        </span>\n\n                                                        {{#compare step \'==\' 1}}\n                                                            Return form registered on {{formatDate ../data.dateSubmitted}}\n                                                        {{else}}\n                                                            {{status}}\n                                                        {{/compare}}\n\n                                                        {{#compare step \'==\' 5}}\n                                            <ul class="statusUl statusUlLast">\n                                                {{else}}\n                                                <ul class="statusUl">\n                                                    {{/compare}}\n\n                                                    {{#each comments}}\n                                                    {{#if visibleForCustomer}}\n                                                    <li>{{comment}}</li>\n                                                    {{/if}}\n                                                    {{/each}}\n                                                </ul>\n\n                                                </p>\n                                        </div>\n\n                                        {{/each}}\n                                        {{/if}}\n                                    </div>\n                                </td>\n                            </tr>\n                        </table>\n\n                    </td>\n                </tr>\n\n                {{/compare}}\n\n            </table>\n        </td>\n    </tr>\n</table>\n</body>\n</html>\n'
```

Comment:
This is really hard to read and can led us to errors. For example, somewhere in this template we have a logic like this:
```
{{else compare data.status "==" \'Denied\'}}
```

Since we decided to have individual templates for, maybe we should remove that. To make it easier to we can try to use a html tag like [this one](https://www.npmjs.com/package/html-template-tag). It also might be helpful to create composable parts for the template. Do you want to give it a try?